src/main/scala/backends/futil/FutilBackend.scala

```diff
@@ -651,7 +651,7 @@ private class FutilBackendHelper {
       "prog",
       List(
         Import("primitives/std.lib"),
-        Component("main", List(), List(), struct.sorted, control)
+        Component(if (c.kernelName == "kernel") "main" else c.kernelName, List(), List(), struct.sorted, control)
```

Comment:
Hmm I dont like doing this. Does anything break in FuTIL if we just let `kernel` be the default name for components?

Reply:
@sgpthomas, as I noted in https://github.com/cucapra/dahlia/pull/324#issuecomment-720163384, I think we would have to change all the FuTIL tests that use Dahlia, and maybe fud itself, to use the right name?

Reply:
when I get back to a computer I'll test this. But I don't think much needs to change. I think we would just have to change the Dahlia frontend tests and then the name that the testbench looks for

Reply:
Sounds legit.

Reply:
yeah, we would have to change a lot of tests. I guess that we can do this for now.